apps/crc_idle.py

```diff
@@ -34,6 +34,7 @@ def __init__(self) -> None:
         self.add_argument('-g', '--gpu', action='store_true', help='list idle resources on the gpu cluster')
         self.add_argument('-m', '--mpi', action='store_true', help='list idle resources on the mpi cluster')
         self.add_argument('-d', '--htc', action='store_true', help='list idle resources on the htc cluster')
+        self.add_argument('-t', '--teach', action='store_true', help='list idle resources on the teach cluster')
```

Comment:
Teach needs to be added to the config on line 21. See lines 53-58 for the resulting impact.

Reply:
Done.